code/modules/mob/living/blood.dm

```diff
@@ -141,29 +152,29 @@
 	if(iscarbon(AM))
 		var/mob/living/carbon/C = AM
 		if(blood_id == C.get_blood_id())//both mobs have the same blood substance
-			if(blood_id == "blood") //normal blood
+			if(blood_id == "blood" || blood_id == "jellyblood") //normal blood
 				if(blood_data["viruses"])
 					for(var/thing in blood_data["viruses"])
 						var/datum/disease/D = thing
 						if((D.spread_flags & DISEASE_SPREAD_SPECIAL) || (D.spread_flags & DISEASE_SPREAD_NON_CONTAGIOUS))
 							continue
 						C.ForceContractDisease(D)
 				if(!(blood_data["blood_type"] in get_safe_blood(C.dna.blood_type)))
-					C.reagents.add_reagent("toxin", amount * 0.5)
-					return 1
+					C.reagents.add_reagent("bonehurtingjuice", amount * 0.5)
```

Comment:
Linzolle's proposed change of adjusting the toxinloss directly would be very much preferable to changing this to bonehurtingjuice. Staminaloss and forcefully spamming "oof", "ouch", and "owie" is not a viable penalty.